internal/wasmdebug/debug.go

```diff
@@ -19,7 +19,7 @@ import (
 //
 //   - moduleName is the possibly empty name the module was instantiated with.
 //   - funcName is the name in the Custom Name section.
-//   - funcIdx is the position in the function index namespace, prefixed with
+//   - funcIdx is the position in the function index space, prefixed with
```

Comment:
bonus points for finding where "index space" could be "index" and mean the same thing